app/controllers/sitemaps_controller.rb

```diff
@@ -41,7 +41,7 @@ def update
                       notice: t(".success")
         end
       else
-        format.html { render :edit }
+        redirect_to edit_planning_application_red_line_boundary_change_path(@planning_application)
```

Comment:
I found edit_planning_application_red_line_boundary_change_validation_request_path but not that one? You could just re-render the page it was on, which I guess is the show page, with a flash message?